src/index.html

```diff
@@ -4,14 +4,111 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalabel=no,
+        initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+      >
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+
+        <label class="forms">
+          Surname:
+          <input type="text" name="surname"
+            minlength="4" maxlength="30">
+        </label>
+        <label class="forms">
+          Name:
+          <input type="text" name="name">
+        </label>
+        <label class="forms">
+          How old are You?
+          <input type="number"
+            name="old"
+            min="1" max="100"
+            value="12">
+        </label>
+        <label class="forms">
+          Full date of birth:
+          <input type="date" name="birth">
+        </label>
+        <label class="forms">
```

Comment:
add one empty line between same level elements
```suggestion
        </label>
        
        <label class="forms">
```